File: ctp-validators/src/main/java/com/commercetools/rmf/validators/RuleSeverity.java
package com.commercetools.rmf.validators;

import com.fasterxml.jackson.annotation.JsonProperty;

public enum RuleSeverity {
    @JsonProperty("error")
    ERROR("error"),
    @JsonProperty("warn")
    WARN("warn"),
    @JsonProperty("info")
    INFO("info");

    private final String severity;

    RuleSeverity(String severity) {
        this.severity = severity;
    }

    public String getSeverity() {
        return severity;
    }
}

File: ctp-validators/src/main/java/com/commercetools/rmf/validators/RuleSeverity.kt
package com.commercetools.rmf.validators

import com.fasterxml.jackson.annotation.JsonProperty

enum class RuleSeverity(val severity: String) {
    @JsonProperty("error")
    ERROR("error"),
    @JsonProperty("warn")
    WARN("warn"),
    @JsonProperty("info")
    INFO("info")
}
